packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -316,6 +333,19 @@ Style guide: components.choice.react
 
 **[View the "Forms" guidelines for additional guidance and best practices.]({{root}}/guidelines/forms/)**
 
+## Checked children and the expose within pattern
```

Comment:
This is great info though it seems a bit disjointed from the props on the usage tab. Is there a way to link to the content from the props?

Reply:
I don't believe so reliably, no. We're using basic anchor links `#usage` and `#guidance` to set the tabs correctly, but without any routing logic. I'd need to have an ID reference for the heading to deep-link to it properly. 

I can add a mention in the `checkedChild` props to see this section in the Guidance tab for further instructions.